package.json

```diff
@@ -2,9 +2,15 @@
   "name": "peaks.js",
   "version": "0.26.0",
   "description": "JavaScript UI component for displaying audio waveforms",
-  "main": "peaks.min.js",
-  "module": "./src/main.js",
+  "main": "./dist/peaks.min.js",
+  "module": "./dist/peaks.esm.min.js",
```

Comment:
I haven't been able to get the esm build to work:

> TypeError: Failed to resolve module specifier "konva/lib/shapes/Line". Relative references must start with either "/", "./", or "../".

Reply:
How did you test it?

Reply:
I use `<script type="module">` in one of the demo pages.

Reply:
Shouldn't `module` still point to `src/main.js`?

Reply:
The way I'm thinking about this is: `module` points to `src/main.js` and people who want an ESM should use rollup or webpack or other bundler to resolve all the dependencies.

Reply:
IIRC, that's the correct way to consume an ESM build with a script tag, I'll have a look at this soon to verify it works.

The `module` entry should point to the ESM build, not the source code, as it is not up to consumers to bundle dependency code. By default, all build tools assume `node_modules` are a set of production-ready dependencies, so all they do is concatenate each dependency bundle into the bundle of your app/library. It's noteworthy that when building a library and externalising its dependencies, all the build-tool does is leave references to whatever each dependency exposes, so when consumed, everything is joined back together without duplications.

Reply:
The ESM build didn't work in the browser because it needs an import map to be able to resolve Konva dependencies, but also other dependencies such as EventEmitter3 and waveform-data aren't ESM format. So the ESM build is only suitable for consumption by build tools.

Reply:
I haven't tested the ESM build in the browser yet, but we are **not** replacing UMD with ESM. We build both to cover everything, so I don't see an issue.

Reply:
It was just an observation.